service/src/main/java/com/ge/predix/acs/service/policy/evaluation/PolicyEvaluationServiceImpl.java

```diff
@@ -80,11 +81,45 @@
     @Override
     public PolicyEvaluationResult evalPolicy(final PolicyEvaluationRequestV1 request) {
         ZoneEntity zone = this.zoneResolver.getZoneEntityOrFail();
-        PolicyEvaluationRequestCacheKey key = new Builder().zoneId(zone.getName()).policySetId("default")
-                .request(request).build();
-        PolicyEvaluationResult result = this.cache.get(key);
+        String uri = request.getResourceIdentifier();
+        String subjectIdentifier = request.getSubjectIdentifier();
+        String action = request.getAction();
+        List<String> policySetsEvaluationOrder = request.getPolicySetsEvaluationOrder();
+
+        if (uri == null || subjectIdentifier == null || action == null) {
+            LOGGER.error(
+                    String.format(
+                            "Policy evaluation request is missing required input parameters: "
+                                    + "resourceURI=[%s] subjectIdentifier=[%s] action=[%s]",
+                            uri, subjectIdentifier, action));
```

Comment:
Everywhere else we use single quote "'" as the delimiter for messages why are we here switching to "["?